test/parallel/test-runner-coverage.js

```diff
@@ -24,12 +24,15 @@ function findCoverageFileForPid(pid) {
 function getTapCoverageFixtureReport() {
```

Comment:
Please add eslint ignore like in:
https://github.com/nodejs/node/blob/6e90fed8c3da4faae9adedd6f1b58cf951382d72/test/parallel/test-runner-coverage.js#L44-L58